File: dmytryshyn/Lab3/Lab3/Lab3/Classes/Ships/Implementations/MediumShip.cs
﻿using MySecondPattern.Classes.Ships;
using MySecondPattern.Classes.Ships.Characteristics;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab3.Classes.Ships.Implementations
{
    public class MediumShip : Ship
    {
        public MediumShip(int id, double fuel, ShipLimitsCharacteristics limitsCharacteristics) : base(id, fuel, limitsCharacteristics)
        {
        }
        public override string ToString()
        {
            return $"The ship {Id} is a medium ship.";
        }
    }
}

File: mukha/lab_3/PortManagementSystem/ships/medium_ship.py
from ships.ship import Ship

class MediumShip(Ship):
    def calculate_fuel(self):
        # Логіка розрахунку пального для середнього корабля
        self.fuel = self.capacity * 0.2  # Приклад логіки розрахунку пального

    def load_container(self, container):
        # Логіка завантаження контейнера на корабель
        pass
